enterprise/frontend/src/embedding-sdk/lib/theme/default-component-theme.ts

```diff
@@ -0,0 +1,12 @@
+import type { MetabaseComponentTheme } from "embedding-sdk";
+
+/**
+ * Default theme options for embedded components.
+ */
+export const DEFAULT_COMPONENT_THEME: MetabaseComponentTheme = {
+  table: {
+    cell: {
+      backgroundColor: "#ffffff",
```

Comment:
Default to white so get good defaults.

![CleanShot 2567-05-13 at 16 48 29@2x](https://github.com/metabase/metabase/assets/4714175/a92f6ec6-8f92-4bfb-bb57-de1543637cb1)